pkg/authprovider/authx/testData/example-auth.enc.yaml

```
@@ -0,0 +1,89 @@
+        - key: ENC[AES256_GCM,data:yc1FH70NQvSd,iv:jXqCn7QMTLPYTwPNFobLYSLdOp+GDG024RSI9DJ9CfY=,tag:+3rBAaRgSj03q53I+9+wjA==,type:str]
+          value: ENC[AES256_GCM,data:01Iq1e144BrHmU0cY/ZzbNKpo/Q=,iv:uwoPqkmL7oNkhOKqbZ9AUEDvcJkvqCvcSzdVeJvrrmE=,tag:9Xw7ACU6gEVUoRxe60CajA==,type:str]
+    # for basic auth session
+    - type: BasicAuth
+      domains:
+        - scanme.sh
+      username: ENC[AES256_GCM,data:FoJCow==,iv:xxQIa91fH2tKJw0aoWUxt8iqBeHswxjxJ8kgwtx8q5w=,tag:nkn/9DhZ8cwJXg1EPX17JQ==,type:str]
+      password: ENC[AES256_GCM,data:oldoEw==,iv:63lG+60DLtiwD5SSg2POnsYVb4v04/XBGYbXkxl8ImM=,tag:2NL5HCBzZ4aW5/0DVkKeqQ==,type:str]
+    # for authorization bearer token
+    - type: BearerToken
+      domains-regex:
+        - .*scanme.sh
```

yamllint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)